docs/testing.md

```diff
@@ -0,0 +1,56 @@
+## Manual Testing Guide
+For non-developing team members to conduct successful manual testing of the platform.
+
+## Mission
+* Work with developers
+* Test new features
+* Test release candidate
+
+## How to Get the Job Done
+### 01) Overview
+Comment observations in GitHub issues that you are working on. If the issue is new/needs special attention, create a new issue for it. If that issue happens that it had been reported earlier, you will be notified by the developers. 
```

Comment:
-1 Some of these sentences appear to not make sense
> If that issue happens that it had been reported earlier, you will be notified by the developers.

could you go through the text and make sure you can understand everything? If something doesn't make sense, feel free to just remove it.